### PR TITLE
Add script that runs tests on dockerized mongodb

### PR DESCRIPTION
**Pull request by rabbitonweb**
This script will
1. clean any existing docker container named some-mongo
2. run new instance of mongodb
3. run tests on that instance
4. clear afterwards

### CODE REVIEW

File: full-it-tests.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+!/bin/bash
```
Comment (paulp) [inline full-it-tests.sh]: Your shebang is missing the leading hash.

**Comment by alissapajer** [comment]
@rabbitonweb LGTM

**Comment by jedesah** [comment]
@rabbitonweb You should squash this down to a single commit and run the merge script (in SlamData/devtools, if you haven't already checked out that repo).

**Comment by alissapajer** [comment]
@rabbitonweb @jedesah This may be a discussion we should have with all of engineering, but sometimes it's nice to have multiple commits in the git history. Each PR being squashed to a single commit doesn't always make sense. If a few distinct things were implemented/fixed in a single PR, then having each of those as a single commit is nice. It makes it easy for someone else to look at a specific change without it being entangled with another change in the diff. And if we ever have to revert a commit, having smaller diffs in each commit is nicer for that reason too.

(And the merge script as it stands does support multiple commits merging into master.)

**Comment by jedesah** [comment]
@alissapajer I agree and I think most of the team would agree as well, but having a formal discussion might prove fruitful nonetheless.

A cursory look at these commits led me to believe that there was not much benefit in having them be separate, which is why I suggested squashing them before merging.

File: full-it-tests.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo "clear some-mongo docker container if exists"
+docker stop some-mongo
+docker rm some-mongo
```
Comment (paulp) [inline full-it-tests.sh]: I'm a little late here, but it's bad practice to be deleting peoples' docker containers without knowing where they came from. This depends on someone not picking the name "some-mongo" for their valuable mongo things, and that's not exactly a random ten character string. I suggest at least putting "quasar" into the container name, like "quasar-some-mongo".

**Pull request closed without merging**